src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/JenkinsService.java

```diff
@@ -192,6 +197,13 @@ protected void addFeedbackToResult(Result result, AbstractBuildResultNotificatio
             result.addFeedbacks(scaFeedbackList);
         }
 
+        final var testwiseCoverageReport = ((TestResultsDTO) buildResult).getTestwiseCoverageReport();
```

Comment:
Since the buildResult is casted multiple times, I think a local variable with the correct static type would be nicer